src/_internals/item/Item.tsx

```diff
@@ -174,6 +160,30 @@ export const Item = (props: ItemProps) => {
           </Text>
         )}
       </span>
+    </Fragment>
+  )
+
+  return (
+    <Tag
+      {...tagProps}
+      onClick={onClick}
+      onFocus={onFocus}
+      onBlur={onBlur}
+      onMouseDown={onMouseDown}
+      className={cc([
+        'kirk-item',
+        {
+          'kirk-item--highlighted': highlighted,
+          'kirk-item--clickable': isClickable,
+          'kirk-item--wrappable': isWrappable,
+          'kirk-item--hideHoverBackground': hideHoverBackground,
+        },
+        className,
+      ])}
+      {...a11yAttrs}
+    >
+      {(isWrappable && <span className="kirk-item-leftWrapper">{leftContent()}</span>) ||
```

Comment:
This new container ensure all left content `leftAddon` + `leftTitle` + `leftBody` + `leftBodyAnnotation` stay on the same line in case of `isWrappable`. We only want right content to wrap.